core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -388,6 +388,15 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
              |It can be set either on the configuration object or via '${SharedBackendConf.PROP_CLOUD_NAME._1}'
              |spark configuration property""".stripMargin)
       }
+
+      if (conf.h2oCluster.isEmpty) {
```

Comment:
What about automatic backend? Shouldn't this condition hold only for manual external backend?

Reply:
Yup, you are right. But I think it does what we need at the moment as it is located in the else block which is executed in case of manual start mode - see line 330 where the if-else starts

Reply:
I see thanks!